python/ql/src/Security/CWE-327/InsecureProtocol.ql

```diff
@@ -0,0 +1,92 @@
+/**
+ * @name Use of insecure SSL/TLS version
+ * @description An insecure version of SSL/TLS has been specified. This may
+ *              leave the connection open to attacks.
+ * @id py/insecure-protocol
+ * @kind problem
+ * @problem.severity warning
+ * @precision high
+ * @tags security
+ *       external/cwe/cwe-327
+ */
+
+import python
+
+FunctionObject ssl_wrap_socket() {
+    result = the_ssl_module().getAttribute("wrap_socket")
+}
+
+ClassObject ssl_Context_class() {
+    result = the_ssl_module().getAttribute("SSLContext")
+}
+
+string insecure_version_name() {
+    // For `pyOpenSSL.SSL`
+    result = "SSLv2_METHOD" or
+    result = "SSLv23_METHOD" or
+    result = "SSLv3_METHOD" or
+    result = "TLSv1_METHOD" or
+    // For the `ssl` module
+    result = "PROTOCOL_SSLv2" or
+    result = "PROTOCOL_SSLv3" or
```

Comment:
Should this include `PROTOCOL_SSLv23` given that you are including `SSLv23_METHOD` for `pyOpenSSL.SSL`